scripts/couchbase-test-helpers/ingest_setup_cluster.py

```diff
@@ -0,0 +1,235 @@
+        f"/index/{cluster_config.search_index_name}"
+    )
+    print("url is", url)
+
+    index_definition = {
+        "type": "fulltext-index",
+        "name": config.search_index_name,
+        "sourceType": "couchbase",
+        "sourceName": config.bucket_name,
+        "planParams": {"maxPartitionsPerPIndex": 1024, "indexPartitions": 1},
+        "params": {
+            "doc_config": {
+                "docid_prefix_delim": "",
+                "docid_regexp": "",
+                "mode": "scope.collection.type_field",
+                "type_field": "type",
+            },
+            "mapping": {
+                "analysis": {},
+                "default_analyzer": "standard",
+                "default_datetime_parser": "dateTimeOptional",
+                "default_field": "_all",
+                "default_mapping": {"dynamic": True, "enabled": False},
+                "default_type": "_default",
+                "docvalues_dynamic": False,
+                "index_dynamic": True,
+                "store_dynamic": True,
+                "type_field": "_type",
+                "types": {
+                    f"{config.scope_name}.{config.collection_name}": {
+                        "dynamic": False,
+                        "enabled": True,
+                        "properties": {
+                            "embedding": {
+                                "dynamic": False,
+                                "enabled": True,
+                                "fields": [
+                                    {
+                                        "dims": 384,
+                                        "index": True,
+                                        "name": "embedding",
+                                        "similarity": "dot_product",
+                                        "type": "vector",
+                                        "vector_index_optimized_for": "recall",
+                                    }
+                                ],
+                            },
+                            "metadata": {"dynamic": True, "enabled": True},
+                            "text": {
+                                "dynamic": False,
+                                "enabled": True,
+                                "fields": [
+                                    {
+                                        "include_in_all": True,
+                                        "index": True,
+                                        "name": "text",
+                                        "store": True,
+                                        "type": "text",
+                                    }
+                                ],
+                            },
+                        },
+                    }
+                },
+            },
+            "store": {"indexType": "scorch", "segmentVersion": 16},
+        },
+        "sourceParams": {},
+    }
+    scope_index_manager = scope.search_indexes()
```

Comment:
Does there need to be a sleep or something between this line and the next line?
I keep getting an error if I try to run this script.
But if I put a breakpoint and wait a while. And then continue it works fine...

Reply:
I also found this issue. I have added appropriate health checks wherever I saw some issues related to this. Please let me know if you still encounter anything like this again